nrich-registry/README.md

```diff
@@ -4,19 +4,22 @@
 
 ## Overview
 
-nrich-registry is a library whose purpose is to make editing of registry entities from client side easier. It transforms JPA entities in a format that client can interpret to create dynamic forms and
-grids for editing of entities without additional implementation on server side. Library provides REST API for searching, creating, updating and deleting entities. Users are only required to provide
+`nrich-registry` is a module whose purpose is to make editing of registry entities from client side easier. It transforms JPA entities in a format that client can interpret to create dynamic forms
+and grids for editing of entities without additional implementation on server side. Module provides REST API for searching, creating, updating and deleting entities. Users are only required to provide
```

Comment:
It should be `tables` instead of `grids` (this should be changed in whole file). Maybe use `administration` instead of `editing`.

Maybe replace `regexes` with `list of regular expressions`. Maybe expand `for including entities` to `for including entites they want to administrate`. 

`and optionally provide...` part should be a new sentence. Maybe something like `Optionally, they can provide display labels...`